File: PRAK402-2210817220011-MeldianaHandayani.py
print("masukkan nilai batas = ", end=' ')
batas= int(input())
for i in range (1, batas):
       if i%2 !=0:
              print('%d '%i,end='')
print("\n")
j= batas
while j>=1:
       if j%2==0:
              print(j , end=' ')
       j-=1

File: PRAK402-2210817220011-MeldianaHandayani.c
#include <stdio.h>
int main(){
int i,x, batas;
printf("masukkan nilai batas = ");
scanf("%d", &batas);
 for(i=1; i<=batas; i+=2){
       printf("%d ",i);
 }
       printf("\n");
 for(x=batas; x>=1 ; x--){ 
      if (x%2 ==0){
          printf("%d ", x);
      }      
 }
          printf("\n");
}
